engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ConstraintValidatorContextImpl.java

```diff
@@ -59,8 +61,10 @@ public ConstraintValidatorContextImpl(
 			ClockProvider clockProvider,
 			PathImpl propertyPath,
 			ConstraintDescriptor<?> constraintDescriptor,
-			Object constraintValidatorPayload) {
+			Object constraintValidatorPayload,
+			ExpressionLanguageFeatureLevel expressionLanguageFeatureLevel) {
 		this.clockProvider = clockProvider;
+		this.defaultExpressionLanguageFeatureLevel = expressionLanguageFeatureLevel;
```

Comment:
Isn't this rather `defaultConstraintExpressionLanguageFeatureLevel`? For non-default constraints, this isn't the default feature level... unless I misunderstand?

Reply:
Yes, exactly. I think I will keep it simple though.

Reply:
In the end, I adjusted it since I introduced another one for custom violations.